client/cmd_test.go

```diff
@@ -42,7 +42,7 @@ func TestValidateCmd(t *testing.T) {
 		{"help flag", []string{"commission", "--help"}, false},       // nolint: misspell
 		{"shorthand help flag", []string{"commission", "-h"}, false}, // nolint: misspell
 		{"flag only, no command provided", []string{"--gas", "1000atom"}, false},
-		{"flag and misspelled command", []string{"--gas", "1000atom", "comission"}, true}, // nolint: misspell
+		{"flag and misspelled command", []string{"--gas", "1000atom", "commission"}, true}, // nolint: misspell
```

Comment:
why did this change? It's intentional.

Reply:
ouch, this was make format result. Thanks

Reply:
The test does not work though @alexanderbez. We should either fix it or remove it.

Reply:
Preferably fix ;-)

Reply:
@jgimeno I think you would just need to replace "comission" with "COMMISSION"

Reply:
True, this test is not working correctly. There was a merge that made it not working as expected.